app/views/people/_biography.html.erb

```diff
@@ -1,10 +1,12 @@
-<%= render "govuk_publishing_components/components/heading", {
-  text: "Biography",
-  id: "biography",
-  margin_bottom: 2,
-} %>
+<section id="biography" class="govuk-!-padding-bottom-9">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Biography",
+    id: "biography",
```

Comment:
```suggestion
```

I don't think we need this if the `section` has an `id` now.

Reply:
👍